extensions/dart/languages/dart/runnables.scm

```diff
@@ -0,0 +1,45 @@
+; Flutter main
+(
+    (
+        (import_or_export
+            (library_import
+                (import_specification
+                    ("import"
+                        (configurable_uri
+                            (uri
+                                (string_literal) @import
+                                (#match? @import "package:flutter/(material|widgets|cupertino).dart")
+                                (#not-match? @import "package:flutter_test/flutter_test.dart")
+                                (#not-match? @import "package:test/test.dart")
+        ))))))
+        (
+            (function_signature
+                name: (_) @run
+            )
+            (#eq? @run "main")
+        )
+        (#set! tag flutter-main)
+    )
+) @flutter-main
+
+; Flutter test main
+(
+    (
+        (import_or_export
+            (library_import
+                (import_specification
+                    ("import"
+                        (configurable_uri
+                            (uri
+                                (string_literal) @import
```

Comment:
```suggestion
                                (string_literal) @_import
```

Unless you want the user to have access to the value of this capture via `ZED_CUSTOM_import` environment variable, you should prefix it so that it is not exposed in env variables of a spawned process.
Same feedback applies to other places using captures (except for `run`, as that one has special treatment and is never exposed).